benchmark/benchmark_runner.cpp

```diff
@@ -145,20 +160,49 @@ double benchmark_runner::calculate_driver_bench_score() {
     return (calculate_read_bench_score() + calculate_write_bench_score()) / 2.0;
 }
 
-void benchmark_runner::print_scores() {
+void benchmark_runner::write_scores() {
     double read = -1;
     double write = -1;
+    const auto end_time = std::chrono::system_clock::now();
```

Comment:
Should probably be set at end of `run_microbenches()` rather than in `write_scores()`.